hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -944,91 +1017,122 @@ used for test and calibration purposes and are not required for runtime
 
-5.   Clear ‘adc\_chnsel\_i’ bus to 0. Note that adc\_chnsel must be
-     cleared to 0 before a new channel is selected.
+5.  Clear ‘adc_chnsel_i’ bus to 0. Note that adc_chnsel must be cleared
+    > to 0 before a new channel is selected.
 
-6.   Repeat steps 3-5 if more channels or more measurements are
-     required
+6.  Repeat steps 3-5 if more channels or more measurements are required
 
-7.   Deactivate the ADC by setting ‘adc\_pd’ to save power.
+7.  Deactivate the ADC by setting ‘adc_pd_i’ to save power.
 
-{{< wavejson >}}
-{
-  signal: [ 
-    {node: '.a..b........', phase:0.2},
-    {name: 'adc_pd_i' ,     wave: '10|..|.....|....|..1'},
-    {name: 'clk_ast_adc_i', wave: 'p.|..|.....|....|...'},
-    {name: 'adc_chnsel_i' , wave: '0.|.3|..04.|....|0..'},
-    {name: 'adc_d_val_o' ,  wave: '0.|..|.1.0.|.1..|.0.'},
-    {name: 'adc_d_o' ,      wave: 'x.|..|.3.x.|.4..|.x.', data: ['ch0', 'ch1', 'ch1']},
-   ],
-
-  edge: [ 'a<->b wakeup time', ] 
-}
-{{< /wavejson >}}
+<u>{{\< wavejson \>}} { signal: \[ {node: '.a..b........', phase:0.2},
+{name: 'adc_pd_i' , wave: '10\|..\|.....\|....\|..1'}, {name:
+'clk_ast_adc_i', wave: 'p.\|..\|.....\|....\|...'}, {name:
+'adc_chnsel_i' , wave: '0.\|.3\|..04.\|....\|0..'}, {name:
+'adc_d\_val_o' , wave: '0.\|..\|.1.0.\|.1..\|.0.'}, {name: 'adc_d\_o' ,
+wave: 'x.\|..\|.3.x.\|.4..\|.x.', data: \['ch0', 'ch1', 'ch1'\]}, \],
+edge: \[ 'a\<-\>b wakeup time', \] } {{\< /wavejson \>}}</u>
 
 # Random Number Generator
 
 AST contains a random number generator that outputs random number
 bitstreams whenever it is enabled. After enabled by the [<u>comportable
-controller</u>](https://github.com/lowRISC/opentitan/blob/master/hw/ip/entropy_src/doc/_index.md)
-through ‘rng\_en\_i’, the AST begins generating multiple independent
-four random bit streams. rng\_b\_o bit streams are valid and can be
-sampled whenever \`rng\_val\_o\` is asserted according to the following
-diagram.
-
-{{< wavejson >}}
-{
-  signal: [
-    {name: 'clk',       wave: 'p.|......|......|......'},
-    {name: 'rng_en',    wave: '01|......|......|......'},
-    {name: 'rng_valid', wave: '0.|10....|10....|10....'},
-    {name: 'rng_b',     wave: 'x.|3.....|4.....|5.....', data: ['es0', 'es1', 'es2']},
-  ]
-}
-{{< /wavejson >}}
-
-The expected rng\_b\_o valid output rate is about 50KHz.
+controller</u>](https://docs.opentitan.org/hw/ip/entropy_src/doc/)
+through ‘rng_en_i’, the AST begins generating multiple independent four
+random bit streams. rng_b\_o bit streams are valid and can be sampled
+whenever \`rng_val_o\` is asserted according to the following diagram.
+
+<u>{{\< wavejson \>}} {signal: \[ {name: 'clk' , wave:
+'p.\|......\|......\|......'}, {name: 'rng_enable' , wave:
+'01\|......\|......\|......'}, {name: 'rng_valid' , wave:
+'0.\|..10..\|..10..\|..10..'}, {name: 'rng_b' , wave:
+'x.\|..3...\|..4...\|..5.....', data: \['es0','es1','es2'\]}, \]} {{\<
+/wavejson \>}}</u>

```

Comment:
All these additional escape characters do not work - the snippets do not render anymore and are just output as a blurb of text. Can you use the previous format for these?

I am referring to:
```
{{\< wavejson \>}} -> {{< wavejson >}}
\| -> |
\] -> ]
```
etc

Also, the previous structure of the wavejson snippets looks more readable.
I suggest to revert to the original formatting...